configs/components/rubygem-nori.rb

```diff
@@ -1,6 +1,11 @@
 component 'rubygem-nori' do |pkg, settings, platform|
-  pkg.version '2.6.0'
-  pkg.md5sum 'dc9c83026c10a3eb7093b9c8208c84f7'
+  if settings[:ruby_version].to_f >= 3.2
+    pkg.version '2.7.1'
+    pkg.md5sum '83952a081b5e86d5aa62943ca9ccf312'
```

Comment:
Did nori drop support for ruby 2.7 in 2.6.0?

Reply:
looks like they dropped support for ruby 2.7 in 2.7.0 - https://rubygems.org/gems/nori/versions/2.6.0